components/org.wso2.carbon.identity.provisioning.connector.google/src/main/java/org/wso2/carbon/identity/provisioning/connector/google/GoogleProvisioningConnector.java

```diff
@@ -231,32 +223,29 @@ protected String createUser(ProvisioningEntity provisioningEntity)
      * @param provisioningEntity
      * @throws IdentityProvisioningException
      */
-    protected void deleteUser(ProvisioningEntity provisioningEntity)
-            throws IdentityProvisioningException {
+    protected void deleteUser(ProvisioningEntity provisioningEntity) throws IdentityProvisioningException {
         boolean isDebugEnabled = log.isDebugEnabled();
```

Comment:
Do we really need this variable?

Reply:
This is used in one other place too. No harm keeping it.